Jenkinsfile

```diff
@@ -12,7 +12,7 @@ def deployerVersion = "master-e4c01702624ac34f2663f894fbdb68a369b09629-588"
 def resources = [
   requestCpu: '1',
   limitCpu: '2',
-  requestMemory: '4Gi',
+  requestMemory: '6Gi',
```

Comment:
Let's see how this works out for us while building. Given [@jasperfurniss updated the memory heap to 4GB](https://github.com/powerhome/playbook/commit/bf3ae54e0e0f4f914d7b2e360cc84b35f6a7aad4), we should need to increase this for headspace.